packages/salesforcedx-vscode-soql/package.json

```diff
@@ -10,15 +10,27 @@
     "url": "https://github.com/forcedotcom/salesforcedx-vscode"
   },
   "version": "50.5.0",
+  "preview": true,
   "publisher": "salesforce",
   "license": "BSD-3-Clause",
   "icon": "images/VSCodeSoql.png",
+  "galleryBanner": {
+    "color": "#ECECEC",
+    "theme": "light"
+  },
   "engines": {
     "vscode": "^1.46.0"
   },
   "categories": [
     "Programming Languages"
   ],
+  "keywords": [
+    "soql",
+    "salesforce",
+    "salesforcedx",
+    "sfdx",
+    "custom editor"
+  ],
```

Comment:
An array of keywords to make it easier to find the extension. These are included with other extension Tags on the Marketplace. This list is currently limited to 5 keywords.